Scripts/Multiplayer/server.gd

```diff
@@ -20,19 +22,31 @@ func _init():
 	LobbySetUnitCommand.register(incoming_commands)
 	ClientRequestedMoveCommand.register(incoming_commands)
 
+func _ready():
+	var request = HTTPRequest.new()
+	add_child(request)
+	request.request_completed.connect(self.onRequestCompleted)
+	request.request("https://api.ipify.org")
```

Comment:
what is this, why

Reply:
discussed on voice
will remove this call from here and add a button that makes a call
with some more readable explanation and more config for the endpoint

Reply:
added fixes

Reply:
Btw the service i used is open source, checks a header in http and writes it back:
https://github.com/rdegges/ipify-api/blob/master/api/get_ip.go#L33C1-L34C1